src/script/script.js

```diff
@@ -0,0 +1,402 @@
+          { className: "Enter", value: "Enter", symbol: false }
+        ],
+        [
+          { className: "ShiftLeft", value: "Shift", symbol: false },
+          { className: "KeyZ", ru: { down: "я", up: "Я", caps: "Я", shiftcaps: "я" }, en: { down: "z", up: "Z", caps: "Z", shiftcaps: "z" }, symbol: true },
+          { className: "KeyX", ru: { down: "ч", up: "Ч", caps: "Ч", shiftcaps: "ч" }, en: { down: "x", up: "X", caps: "X", shiftcaps: "x" }, symbol: true },
+          { className: "KeyC", ru: { down: "с", up: "С", caps: "С", shiftcaps: "с" }, en: { down: "c", up: "C", caps: "C", shiftcaps: "c" }, symbol: true },
+          { className: "KeyV", ru: { down: "м", up: "М", caps: "М", shiftcaps: "м" }, en: { down: "v", up: "V", caps: "V", shiftcaps: "v" }, symbol: true },
+          { className: "KeyB", ru: { down: "и", up: "И", caps: "И", shiftcaps: "и" }, en: { down: "b", up: "B", caps: "B", shiftcaps: "b" }, symbol: true },
+          { className: "KeyN", ru: { down: "т", up: "Т", caps: "Т", shiftcaps: "т" }, en: { down: "n", up: "N", caps: "N", shiftcaps: "n" }, symbol: true },
+          { className: "KeyM", ru: { down: "ь", up: "Ь", caps: "Ь", shiftcaps: "ь" }, en: { down: "m", up: "M", caps: "M", shiftcaps: "m" }, symbol: true },
+          { className: "Comma", ru: { down: "б", up: "Б", caps: "Б", shiftcaps: "б" }, en: { down: ",", up: "<", caps: ",", shiftcaps: "<" }, symbol: true },
+          { className: "Period", ru: { down: "ю", up: "Ю", caps: "Ю", shiftcaps: "ю" }, en: { down: ".", up: ">", caps: ".", shiftcaps: ">" }, symbol: true },
+          { className: "Slash", ru: { down: ".", up: ",", caps: ".", shiftcaps: "," }, en: { down: "/", up: "?", caps: "/", shiftcaps: "?" }, symbol: true },
+          { className: "ArrowUp", value: "▲", symbol: false },
+          { className: "ShiftRight", value: "Shift", symbol: false },
+        ],
+        [
+          { className: "ControlLeft", value: "Ctrl", symbol: false },
+          { className: "MetaLeft", value: "Win", symbol: false },
+          { className: "AltLeft", value: "Alt", symbol: false },
+          { className: "Space", value: "", symbol: false },
+          { className: "AltRight", value: "Alt", symbol: false },
+          { className: "ContextMenu", value: "", symbol: false },
+          { className: "ArrowLeft", value: "◄", symbol: false },
+          { className: "ArrowDown", value: "▼", symbol: false },
+          { className: "ArrowRight", value: "►", symbol: false },
+          { className: "ControlRight", value: "Ctrl", symbol: false }
+        ]
+      ];
+  }
+
+  setRootElement() {
+    this.rootElement = document.createElement('div');
+    this.rootElement.classList.add('virtual-keyboard-wrapper');
+    document.body.prepend(this.rootElement);
+  }
+  ///////////////////////////////////////////////////////////////
+  generateHeader() {
+    if (!this.rootElement) this.setRootElement();
```

Comment:
Why do you check for rootElement in methods? You could set in in Constructor

Reply:
I forgot to remove it all. Fixed.